build_engine_test_images/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -z "$TF_VAR_build_engine_aws_access_key" ]]; then
```

Comment:
I think terraform would check variables are set or not automatically?

Reply:
This part just want to early check when script beginning.